### PR TITLE
feat(suite): coinjoin summary section UI

### LINKED ISSUE #6102: Distinguish CoinJoin state amounts
CoinJoin amounts might appear in 3 states:
- `Not private` - UTXOs with anonymity bellow desired anonymity level
- `Anonymising` - UTXOs in process of anonymisation
- `Private` - Already anonymised UTXOs to desired anonymity level or higher

Image

- `Anonymize` button leads to https://github.com/trezor/trezor-suite/issues/6101
- `Anonymising` amount is visible only if there are some funds in running CoinJoin

Image

- `Anonymize` button disappears when are all funds anonymised to desired anonymity level

### PR DESCRIPTION
**Pull request by dahaca**
## Description

Implementing of the UI for coinjoin summary. Since there is no way to distinguish between anonymized and anonymized amounts for now the values are placeholders. The file structure and logic will be finalized once @komret's changes are merged.

## Related Issue

Resolve #6102

## Screenshots:

image

### CODE REVIEW

File: packages/suite/src/support/messages.ts [hunk 1]
```diff
@@ -6884,4 +6884,32 @@ export default defineMessages({
         id: 'TR_FEE_RATE_CHANGED',
         defaultMessage: 'Fee rate has changed to complete transaction.',
     },
+    TR_MY_COINS: {
+        id: 'TR_MY_COINS',
+        defaultMessage: 'My coins',
+    },
+    TR_ANONYMIZING: {
+        id: 'TR_ANONYMIZING',
+        defaultMessage: 'Anonimyzing',
```
Comment (komret) [inline packages/suite/src/support/messages.ts:6889]: Should read "Anonymizing" (also line 6897 and 6905).

File: packages/suite/src/components/wallet/CoinjoinSummary/AccountSummarySection/BalanceSection.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+const AnonymizeButton = styled(Button)`
+    justify-content: space-between;
+    width: 154px;
+    height: 46px;
+    padding: 9px 18px;
+`;
+
+interface BalanceSectionProps {
+    onAnonimize: () => void; // TEMPORARY
```
Comment (komret) [inline packages/suite/src/components/wallet/CoinjoinSummary/AccountSummarySection/BalanceSection.tsx:23]: Should read "onAnonymize". Repeats on other places as well.

File: packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    headerIcon,
+    value,
+    symbol,
+    color,
+    note,
+    className,
+}: CryptoAmountWithHeaderProps) => (
+    <Container className={className}>
+        <Header>
+            {headerIcon && <HeaderIcon icon={headerIcon} size={12} />} {header}
```
Comment (komret) [inline packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx:61]: I think the icons are too small, especially the closed eye is hard to recognize. You can crop the icons in https://svgcrop.com/ (and ideally also optimize them in https://iconly.io/tools/svg-cleaner) or increase the size here.

File: packages/suite/src/components/wallet/CoinjoinSummary/AccountSummarySection/AnonymityIndicator.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    border-radius: 8px;
+    background: ${({ theme }) => theme.STROKE_GREY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    text-align: right;
+`;
+
+const UsersIcon = styled(Icon)`
+    path {
+        fill: none;
```
Comment (komret) [inline packages/suite/src/components/wallet/CoinjoinSummary/AccountSummarySection/AnonymityIndicator.tsx:21]: Please convert the SVG from strokes to fills in https://iconly.io/tools/svg-convert-stroke-to-fill, then you can remove this style.

File: packages/suite/src/components/wallet/CoinjoinSummary/AccountSummarySection/FundsPrivacyBreakdown.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+    const isSessionRunning = true;
+
+    return (
+        <Container isSessionRunning={isSessionRunning}>
+            <CryptoAmountWithHeader
+                header={<Translation id="TR_NOT_PRIVATE" />}
+                headerIcon="CROSS"
+                value={privateAmount}
+                symbol="btc"
```
Comment (komret) [inline packages/suite/src/components/wallet/CoinjoinSummary/AccountSummarySection/FundsPrivacyBreakdown.tsx:30]: Could use `account.symbol` to differentiate between testnet and mainnet. However, some values might not fit one line, e.g. "sat REGTEST".

**Review by komret — CHANGES_REQUESTED** [review]
Good stuff!

File: packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        </Header>
+
+        <CryptoAmount value={value} symbol={symbol} color={color} />
```
Comment (komret) [inline packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx:64]: Could this component also convert the amount to the correct unit? E.g. I provide `account.balance` and it converts it from `sat` to `BTC`.

File: packages/suite/src/support/messages.ts (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/support/messages.ts:6889]: I made this mistake like 100 times during development, the word is cursed, it's literally anonymizing itself

File: packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx (hunk 6 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx:64]: It is converting but accepts amounts in `BTC` after recent changes by Marek, I will change it soon.

File: packages/suite/src/components/wallet/CoinjoinSummary/AccountSummarySection/FundsPrivacyBreakdown.tsx (hunk 5 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/CoinjoinSummary/AccountSummarySection/FundsPrivacyBreakdown.tsx:30]: Yes, of course it should be `account.symbol `, all the symbol formatting is done inside `FormattedCryptoAmount`

File: packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx (hunk 3 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx:61]: That size is the same as on designs and matches the font size

File: packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx (hunk 6 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx:64]: @marekrjpolak could you confirm that you wanted as much as possible to use `sat`? Essentially, going forward, it's preferred to use `BTC` and like only for displaying for users?

File: packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx (hunk 3 above)
Comment (komret) [reply packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx:61]: d41b0aaf21d84898fbca2cb56801c986

**Review by komret — APPROVED** [review]

**Pull request merged**